### PR TITLE
AccountController and group routes

### PR DESCRIPTION
**Pull request by ManojKiranA**
This PR Fixes following

1. Error on 500 status code
From
```
// 500 error
Route::get('500', function () {
    echo $fail;
});
```
To
```
// 500 error
Route::get('500', function () {
    abort(500);
});
```
2. Group all the route middlewares

while analysing the `web.php` file there is a `auth` middleware in most of the web routes and its declared in each of the routes so its `grouped together and added the auth middleware`

3. Added AccountsController
in the routes the AccountsController is used but its never created.

### CODE REVIEW

File: app/Http/Controllers/AccountsController.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<?php
```
Comment (reinink) [inline app/Http/Controllers/AccountsController.php]: This isn't needed...since we don't actually have any account management in Ping CRM. Please remove this file, and also remove the corresponding routes.

File: app/Providers/AppServiceProvider.php [hunk 2]
```diff
@@ -21,6 +22,8 @@ class AppServiceProvider extends ServiceProvider
     public function boot()
     {
         Date::use(CarbonImmutable::class);
+        Schema::defaultStringLength(191);
```
Comment (reinink) [inline app/Providers/AppServiceProvider.php]: Why is this required?

File: routes/web.php [hunk 3]
```diff
@@ -1,4 +1,5 @@
 <?php
+use Illuminate\Support\Facades\Artisan;
```
Comment (reinink) [inline routes/web.php]: This import isn't necessary.

File: routes/web.php [hunk 4]
```diff
@@ -17,48 +18,51 @@
 Route::post('logout')->name('logout')->uses('Auth\LoginController@logout');
 
 // Dashboard
-Route::get('/')->name('dashboard')->uses('DashboardController')->middleware('auth');
+Route::get('/')->name('dashboard')->uses('DashboardController');
 
-// Accounts
```
Comment (reinink) [inline routes/web.php]: Please remove all the "accounts" routes.

File: routes/web.php [hunk 5]
```diff
@@ -17,48 +18,51 @@
...
-Route::get('accounts')->name('accounts')->uses('AccountsController@index')->middleware('remember', 'auth');
-Route::get('accounts/create')->name('accounts.create')->uses('AccountsController@create')->middleware('auth');
-Route::post('accounts')->name('accounts.store')->uses('AccountsController@store')->middleware('auth');
-Route::get('accounts/{account}/edit')->name('accounts.edit')->uses('AccountsController@edit')->middleware('auth');
-Route::put('accounts/{account}')->name('accounts.update')->uses('AccountsController@update')->middleware('auth');
-Route::delete('accounts/{account}')->name('accounts.destroy')->uses('AccountsController@destroy')->middleware('auth');
-Route::put('accounts/{account}/restore')->name('accounts.restore')->uses('AccountsController@restore')->middleware('auth');
+Route::group(['middleware' => ['auth']],function(){
```
Comment (reinink) [inline routes/web.php]: I'd prefer to leave the routes as they were, without the the groups. Please revert that change.

File: routes/web.php [hunk 6]
```diff
@@ -17,48 +18,51 @@
...
-Route::delete('contacts/{contact}')->name('contacts.destroy')->uses('ContactsController@destroy')->middleware('auth');
-Route::put('contacts/{contact}/restore')->name('contacts.restore')->uses('ContactsController@restore')->middleware('auth');
 
 // Reports
-Route::get('reports')->name('reports')->uses('ReportsController')->middleware('auth');
+Route::get('reports')->name('reports')->uses('ReportsController');
 
 // 500 error
 Route::get('500', function () {
-    echo $fail;
```
Comment (reinink) [inline routes/web.php]: This is an intentional bug...to illustrate how error handling is done in Inertia. Please revert change.

File: package-lock.json [hunk 7]
```diff
@@ -1411,7 +1411,6 @@
             "version": "0.18.0",
```
Comment (reinink) [inline package-lock.json]: Please remove package-lock.json from this PR.

File: app/Http/Controllers/AccountsController.php (hunk 1 above)
Comment (ManojKiranA) [reply app/Http/Controllers/AccountsController.php]: i have added the `AccountsController` because the routes are defined.ok  i will revert that. nope

Comment (reinink) [reply app/Http/Controllers/AccountsController.php]: Understood. That was there still from a previous project that I use as the basis for Ping CRM. You can safely remove.

File: app/Providers/AppServiceProvider.php (hunk 2 above)
Comment (ManojKiranA) [reply app/Providers/AppServiceProvider.php]: i have added because my maria db version

For example

`$table->string('first_name', 25);`

in this migration if i removed `25` i wont takes the default string length but i will revert that too.nope

File: routes/web.php (hunk 3 above)
Comment (ManojKiranA) [reply routes/web.php]: done sir

File: routes/web.php (hunk 4 above)
Comment (ManojKiranA) [reply routes/web.php]: done

File: routes/web.php (hunk 5 above)
Comment (ManojKiranA) [reply routes/web.php]: there is no issue with group routes and its does not change anything.i have done this because you have added `auth` middleware in except `login and logout` so i have grouped together to add the middleware and makes the code clean. 

If you dont like that i can 
**add the middleware in _construct method of controller** 
or
 **do you need to revert it to original??**

File: routes/web.php (hunk 6 above)
Comment (ManojKiranA) [reply routes/web.php]: done

File: package-lock.json (hunk 7 above)
Comment (ManojKiranA) [reply package-lock.json]: done sir

File: app/Http/Controllers/AccountsController.php (hunk 1 above)
Comment (ManojKiranA) [reply app/Http/Controllers/AccountsController.php]: ok sir done

**Comment by ManojKiranA** [comment]
Sir Do check and inform me if there is any changes to be done

File: .gitignore [hunk 8]
```diff
@@ -13,4 +13,4 @@
 Homestead.json
 Homestead.yaml
 npm-debug.log
-yarn-error.log
+yarn-error.log
```
Comment (reinink) [inline .gitignore]: Please revert this change.

File: package-lock.json [hunk 9]
```diff
@@ -1,10248 +0,0 @@
-{
```
Comment (reinink) [inline package-lock.json]: Please don't delete this file. Simply revert the changes.

File: routes/web.php (hunk 4 above)
Comment (reinink) [reply routes/web.php]: Please do not delete the dashboard route.

File: routes/web.php (hunk 5 above)
Comment (reinink) [reply routes/web.php]: I would prefer to leave it as it was.

File: package-lock.json (hunk 9 above)
Comment (ManojKiranA) [reply package-lock.json]: ok sir will do it

**Comment by ManojKiranA** [comment]
check now sir its fixed

**Comment by reinink** [comment]
@ManojKiranA I really appreciate you finding these issues, but since this PR still has a lot of changes I don't really want, I've simply made those changes to the repo myself. I'm going to close this PR...but again, I appreciate your efforts! 👏

**Comment by ManojKiranA** [comment]
ok sir no problem
